# functions1 hint

この`main`関数は`call_me`関数があると思って呼び出していますが、実際にはありません。

その`call_me`関数は引数を取らず、値を返すこともしないでしょう。

まるで`main`関数のように。

<!---
This main function is calling a function that it expects to exist, but the
function doesn't exist. It expects this function to have the name `call_me`.
It expects this function to not take any arguments and not return a value.
Sounds a lot like `main`, doesn't it?
--->
